### PR TITLE
[Gallery Layouts] mobile gallery options

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Introduces Mobile controls for the gallery options
image
image
image

# Why

gives the option to use
- dynamic layout
- Gallery layout
on mobile

https://skype.visualstudio.com/SPOOL/_workitems/edit/3417192
# How Tested

tested locally e2e to come

### CODE REVIEW

File: packages/react-composites/tests/browser/snapshots/beta/tests/browser/call/hermetic/CallControls.test.ts-snapshots/call-control-new-experience-custom-button-Mobile-Android-Landscape-linux.png
Comment (JamesBurnside) [inline packages/react-composites/tests/browser/snapshots/beta/tests/browser/call/hermetic/CallControls.test.ts-snapshots/call-control-new-experience-custom-button-Mobile-Android-Landscape-linux.png:1]: This test is called custom button but I don't see one

File: packages/react-composites/tests/browser/snapshots/beta/tests/browser/callwithchat/live/CallWithChatComposite.test.ts-snapshots/call-with-chat-more-drawer-dtmf-dialpad-Mobile-Android-Portrait-linux.png
Comment (JamesBurnside) [inline packages/react-composites/tests/browser/snapshots/beta/tests/browser/callwithchat/live/CallWithChatComposite.test.ts-snapshots/call-with-chat-more-drawer-dtmf-dialpad-Mobile-Android-Portrait-linux.png:1]: huh, looks like we should proabbly be dismissing the menu when opening dialpad

**Comment by JamesBurnside** [comment]
How come image in PR doesn't have focused content view or speaker view?
image

File: packages/react-composites/tests/browser/snapshots/beta/tests/browser/call/hermetic/CallControls.test.ts-snapshots/call-control-new-experience-custom-button-Mobile-Android-Landscape-linux.png
Comment (dmceachernmsft) [reply packages/react-composites/tests/browser/snapshots/beta/tests/browser/call/hermetic/CallControls.test.ts-snapshots/call-control-new-experience-custom-button-Mobile-Android-Landscape-linux.png:1]: I think the introduction of the gallery options scrolled it down?

Comment (edwardlee-msft) [reply packages/react-composites/tests/browser/snapshots/beta/tests/browser/call/hermetic/CallControls.test.ts-snapshots/call-control-new-experience-custom-button-Mobile-Android-Landscape-linux.png:1]: yeah, i notice there is a scroll bar on the right.

File: packages/react-composites/tests/browser/snapshots/beta/tests/browser/callwithchat/live/CallWithChatComposite.test.ts-snapshots/call-with-chat-more-drawer-dtmf-dialpad-Mobile-Android-Portrait-linux.png
Comment (edwardlee-msft) [reply packages/react-composites/tests/browser/snapshots/beta/tests/browser/callwithchat/live/CallWithChatComposite.test.ts-snapshots/call-with-chat-more-drawer-dtmf-dialpad-Mobile-Android-Portrait-linux.png:1]: Good catch, this is probably out of scope for this feature though. Chalk it up as a bug.

**Review by edwardlee-msft — APPROVED** [review]

**Comment by dmceachernmsft** [comment]
> How come mobile doesn't have focused content view or speaker view? image

You will find that updated post bug bash 😊
image

File: packages/react-composites/tests/browser/snapshots/beta/tests/browser/call/hermetic/CallControls.test.ts-snapshots/call-control-new-experience-custom-button-Mobile-Android-Landscape-linux.png
Comment (JamesBurnside) [reply packages/react-composites/tests/browser/snapshots/beta/tests/browser/call/hermetic/CallControls.test.ts-snapshots/call-control-new-experience-custom-button-Mobile-Android-Landscape-linux.png:1]: I don't see a custom button though

**Review by JamesBurnside — APPROVED** [review]

**Pull request merged**